packages/mdc-chips/chip/foundation.ts

```diff
@@ -71,9 +71,7 @@ export class MDCChipFoundation extends MDCFoundation<MDCChipAdapter> {
     };
   }
 
-  /**
```

Comment:
is there any bug created on `dts-bundler`?

The comment syntax looks correct to me. We might come across this issue in future if we add back this comment style.

Reply:
Agreed that there's nothing wrong with the comment. I couldn't find anything in the `dts-bundler` issue tracker and the project seems mostly dead. I think that the correct approach here would be to use the compiler to generate the files during compilation, rather than depending on a third party tool.

Reply:
Thanks @crisbeto!

Created Issue #5580 to address this issue.